docs/QSG/docker.md

```diff
@@ -1,212 +1,102 @@
+
+## Hardware and Software Prerequisites
+### Hardware Prerequisites
+- 5GB of DDR and 
+- 16GB of storage.
+
+### Software Prerequisites
+- BIOS
+	- Enable VT-d 
+- To build and deploy the Docker images, `docker` 
+	- optionally `docker-compose`
+- Hugepages must be enabled
+	- At least, 4096 pages of 2048kB should be available 
+- Internet access for the following repositories
+	- [Docker Hub](https://hub.docker.com/) 
+	- [Rocky Linux](https://rockylinux.org/) official repositories.  
+	- [hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) enabled
+		-Linux kernel feature shall be enabled on the docker host. At least, 4096 pages of 2048kB should be available. 
+
+### Hardware and software configuration used in this QSG
+- Server Board: Wolf Pass
+- CPU: Cascade Lake (2)
+- OS: Ubuntu 20.0.4LTE. fresh Install
+- Memory: 192GB 
+- PMEM:(12)x16GB (not required for this QSG)
+- BIIOS: VT-d enable
+
+## Configuring Hugepages
+Additional information in Hugepages is located in the administration portion of this guide <need link> and at (kernel.org)[https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt]
+	
+### Determining Huge pages
+The default size of a huge page, the number of available huge pages, etc. could be found with the following command: 
+- Ubuntu- ```$ cat /proc/meminfo | grep -e "^Huge" ```
+If the pages are not enabled, you will see something like this:
 ```
-
-The default size of a huge page, the number of available huge pages, etc. could be found with the
-following command:
-
-```bash
-$ cat /proc/meminfo | grep -e "^Huge"
+HugePages_Total:       0
+HugePages_Free:        0
+HugePages_Rsvd:        0
+HugePages_Surp:        0
+Hugepagesize:       2048 kB
+Hugetlb:               0 kB
 ```
 
-The platform was tested and validated with the
-[rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) and
-[centos:centos8](https://hub.docker.com/_/centos) official docker images.  However other RHEL-like
-distributions should be supported.
-
-!!! warning
-    Some distributions are not yet well supported such as
-    [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the
-    management of hugepages with the [spdk](https://spdk.io/) library.
-
-
+#### Note: 
+Huge pages are supported by all containerized Linux distributions. However support of huge pages 
+with DAOS and its dependencies re not working for some Linux distribution.only the following distributions have
+been validated:
+- [rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) 
+- [centos:centos8](https://hub.docker.com/_/centos) official docker images.
+The following Linux docker distributions are known not to support hugepages at this time:
+- [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux)
 
 ### Configuring HugePages
```

Comment:
ditto.

Reply:
confused, but if its still a problem after the changes I made today let me know